CapMan.Raylib/CapManRenderer.cs

```diff
@@ -49,6 +49,8 @@ public void Render(PlayerActor capman, int left, int top)
         Sprite.FlipX = capman.CurrentDirection == Direction.Right ? true : false;
         int x = (int)(capman.Position.X * BoardRenderer.CellSize) + BoardRenderer.CellSize / 2;
         int y = (int)(capman.Position.Y * BoardRenderer.CellSize) + BoardRenderer.CellSize / 2;
+        x = (int)(x * capman.Speed);
+        y = (int)(y * capman.Speed);
         Sprite.Draw(left + x, top + y, Color.White);
```

Comment:
Something feels off to me about X and Y being adjusted by the player's speed. Why is this necessary?